Fix the gumby documentation and remove references to das4

pvanita opened the pull request:
modified:   index.rst
	new file:   quickoverview.rst
	modified:   tutorials/experiment_tutorial_3.rst
	renamed:    tutorials/synchronized_instances_das5.conf -> tutorials/synchronized_instances_das.conf
	renamed:    ../scripts/das4_node_run_job.sh -> ../scripts/das_node_run_job.sh
	renamed:    ../scripts/das4_reserve_and_run.sh -> ../scripts/das_reserve_and_run.sh
	renamed:    ../scripts/das4_setup.sh -> ../scripts/das_setup.sh

Fixes partially #501

kozlovsky committed:
Fix DHT log error when only one instance is launched

kozlovsky committed:
Fix discovery community settings

kozlovsky committed:
Fix exception message text in IPv8OverlayExperimentModule.ipv8_provider

kozlovsky committed:
Add debug message when installing Python in build_virtualenv.sh

pvanita committed:
modified:   gumby/scenario.py
Remove scenario language for_loop feature

for_loops code has been removed from scenario.py.
for_loops documentation and testscript  have been removed.

devos50 committed:
Removed gmpy2 and dependencies

kozlovsky committed:
Make Gumby compatible with the current version of Tribler

kozlovsky committed:
Move stuff around

kozlovsky committed:
Fix _stats_file AttributeError introduced in c5aa64a7

kozlovsky committed:
Remove unused GumbyTriblerConfig

kozlovsky committed:
Split tribler_config and gumby_config as a preparation step to a cleaner architecture

kozlovsky committed:
Satisfy linter

kozlovsky committed:
Configurable start of Tribler's components

kozlovsky committed:
Fix extracting PGRP value from the stat file

kozlovsky committed:
Satisfy linter

kozlovsky committed:
Fix tribler_idle_run

kozlovsky committed:
Remove obsolete callback `start_bootstrap_download`

kozlovsky committed:
Add pyipv8 to the list of requirements in build_virtualenv.sh

kozlovsky committed:
Remove handling of obsolete directories tribler/src/tribler-common and tribler/src/pyipv8

pvanita committed:
Fix the gumby documentation and remove references to das4

	modified:   index.rst
	new file:   quickoverview.rst
	modified:   tutorials/experiment_tutorial_3.rst
	renamed:    tutorials/synchronized_instances_das5.conf -> tutorials/synchronized_instances_das.conf
	renamed:    ../scripts/das4_node_run_job.sh -> ../scripts/das_node_run_job.sh
	renamed:    ../scripts/das4_reserve_and_run.sh -> ../scripts/das_reserve_and_run.sh
	renamed:    ../scripts/das4_setup.sh -> ../scripts/das_setup.sh
	new file:   docs/MdVGumbyOverview.png
	modified:   docs/quickoverview.rst

Resolves: #501

synctext commented on docs/quickoverview.rst:
```diff
@@ -0,0 +1,18 @@
+
+The interaction between the researcher and the gumby framework can be presented als follows:
+
+.. image:: MVosOveriewA.png
```
Is this picture already in the repo?

devos50 replied:
Indeed, the picture should be included 👍

devos50 commented:
ok to test

devos50 commented:
The DAS experiment seems to fail, see [here](https://jenkins-ci.tribler.org/job/gumby/job/PR_experiment_3_DAS5/192/console).

devos50 commented:
retest this please

devos50 commented:
@pvanita it seems that the file you included is not the correct one: it includes information about 'wallet' and 'validator' nodes. That image was made for our blockchain experiments but does not accurately reflect the capabilities of Gumby to the readers of the documentation.

Could you please add the right figure? This figure should be based on the one I've sent to your earlier (I think by mail) but some text needs to be changed.

I'll review the remainder of the PR a bit later.

devos50 commented on docs/quickoverview.rst:
```diff
@@ -0,0 +1,46 @@
+`DAS5  <https://www.cs.vu.nl/das5/clusters.shtml>`_.
+`DAS6  <https://www.cs.vu.nl/das6/clusters.shtml>`_.
+
+The interaction between the researcher and the gumby framework can be presented als follows:
```
Typo: als -> as

devos50 commented on docs/quickoverview.rst:
```diff
@@ -0,0 +1,46 @@
+   :width: 600
+
+
+Installation
```
Please make this a header.

devos50 commented on docs/quickoverview.rst:
```diff
@@ -0,0 +1,46 @@
+   $ git clone https://github.com/tribler/gumby
+
+
+In the Tutorials section you will find information on how to create your first Gumby experiment.
```
Can you add a link/reference to these tutorials?

devos50 commented on docs/tutorials/experiment_tutorial_3.rst:
```diff
@@ -1,41 +1,41 @@
-The maximum number of compute nodes in each cluster can be found `here <https://www.cs.vu.nl/das5/clusters.shtml>`_.
+Additionally, there are a few configuration options that are specific to DAS related experiments.
+The ``node_amount`` configuration option indicates how many DAS compute nodes are used.
+The maximum number of compute nodes in each cluster can be found at the DAS sites listed in the Quick Overview.
```
Can you add a link to the quick overview?

devos50 commented on docs/quickoverview.rst:
```diff
@@ -0,0 +1,46 @@
+
+Prior to installing Gumby install the required dependencies for basic tests.
+Those are the software packages: 
+python-psutil
```
I don't think it's necessary to explicitly list the required packages since they are already part of the `apt-get install` command. The reader would know the requirements by looking at the command below.

devos50 commented:
One more note: the DAS5 experiment fails since I had to restore it to get the correct test output for #515.

Also, could you try to rebase your PR on top of the base branch?

pvanita commented:
Hi Martijn,
I understand that as long as the "name changing" has not been merged you will have to keep the original test to get other PR's through the testing.
I have to reread the "rebasing"mechanism to be able to do it correctly.
I will let you know when it is done.
Regards and thanks for your help.

Paulo

________________________________
From: Martijn de Vos ***@***.***>
Sent: Wednesday, March 23, 2022 9:09 AM
To: Tribler/gumby
Cc: Paulo Anita; Mention
Subject: Re: [Tribler/gumby] Fix the gumby documentation and remove references to das4 (PR #514)


One more note: the DAS5 experiment fails since I had to restore it to get the correct test output for #515<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Tribler_gumby_pull_515&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=cEsBxV1c8zRwWRWa7_aQCe-xc9D7GU-fk4YXRwwvxOc&e=>.

Also, could you try to rebase your PR on top of the base branch?

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Tribler_gumby_pull_514-23issuecomment-2D1076059563&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=TlCzunfUb6MYbMKmkd6f-9dTX1_ifrJOmnqfuRL0bwc&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AJNQSJTUEVTZJNBK4PI3OOLVBLGSLANCNFSM5QBZZRFQ&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=lsZIawqB51bMkK1jyi_QOZ2Jk-Ir9km-vErla7JXq3I&e=>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

pvanita committed:
Revert "Merge pull request #508 from kozlovsky/tribler_fixes"

This reverts commit b9d9d9e6fa9f4545b0bb2d8d51d9191662f69ec3, reversing
changes made to 6e3b08fd1c54224a0b5f8c09557d90e391c16695.

pvanita commented:
Hello Martijn,

After a "git rebase -i " and pushing with option -f to the pvanita/gumby FixDocs branch  I noticed in the history that other commits has been changed to ( example )

kozlovsky<https://github.com/pvanita/gumby/commits?author=kozlovsky> authored and pvanita<https://github.com/pvanita/gumby/commits?author=pvanita> committed 11 minutes ago


I left those comments unchanged when rebasing.  How can I undo this?

Or can those modified history entries be left out when merging the PR?


Regards,

Paulo

________________________________
From: Paulo Anita
Sent: Wednesday, March 23, 2022 9:30:59 AM
To: Tribler/gumby; Tribler/gumby
Cc: Mention
Subject: Re: [Tribler/gumby] Fix the gumby documentation and remove references to das4 (PR #514)



Hi Martijn,
I understand that as long as the "name changing" has not been merged you will have to keep the original test to get other PR's through the testing.
I have to reread the "rebasing"mechanism to be able to do it correctly.
I will let you know when it is done.
Regards and thanks for your help.

Paulo

________________________________
From: Martijn de Vos ***@***.***>
Sent: Wednesday, March 23, 2022 9:09 AM
To: Tribler/gumby
Cc: Paulo Anita; Mention
Subject: Re: [Tribler/gumby] Fix the gumby documentation and remove references to das4 (PR #514)


One more note: the DAS5 experiment fails since I had to restore it to get the correct test output for #515<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Tribler_gumby_pull_515&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=cEsBxV1c8zRwWRWa7_aQCe-xc9D7GU-fk4YXRwwvxOc&e=>.

Also, could you try to rebase your PR on top of the base branch?

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Tribler_gumby_pull_514-23issuecomment-2D1076059563&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=TlCzunfUb6MYbMKmkd6f-9dTX1_ifrJOmnqfuRL0bwc&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AJNQSJTUEVTZJNBK4PI3OOLVBLGSLANCNFSM5QBZZRFQ&d=DwMCaQ&c=XYzUhXBD2cD-CornpT4QE19xOJBbRy-TBPLK0X9U2o8&r=CJDgNjScSOB2FNitfudA3ojnB8KnkjkMKmfLBFZu3tM&m=Z1aTcvj_csiyeDTkyp7h2xN8Vx60QLEHeL5Mav4hUDk&s=lsZIawqB51bMkK1jyi_QOZ2Jk-Ir9km-vErla7JXq3I&e=>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

devos50 commented:
@pvanita it indeed seems that other commits are now part of this PR. I believe that they will vanish when merging, but nonetheless it's not good practice to leave them in this PR like they currently are.

When you want to rebase on the target branch, an interactive rebase is usually not necessary. Assuming that `upstream` is your remote that points to https://github.com/tribler/gumby, running `git rebase upstream/devel` _should_ fix the issue (note that this is not an interactive rebase). Could you try to do this?

pvanita committed:
Fix the gumby documentation and remove references to das4

        modified:   index.rst
        new file:   quickoverview.rst
        modified:   tutorials/experiment_tutorial_3.rst
        renamed:    tutorials/synchronized_instances_das5.conf -> tutorials/synchronized_instances_das.conf
        renamed:    ../scripts/das4_node_run_job.sh -> ../scripts/das_node_run_job.sh
        renamed:    ../scripts/das4_reserve_and_run.sh -> ../scripts/das_reserve_and_run.sh
        renamed:    ../scripts/das4_setup.sh -> ../scripts/das_setup.sh
        new file:   docs/MdVGumbyOverview.png
        modified:   docs/quickoverview.rst

    Resolves: #501
